docker-compose.yml

```diff
@@ -410,6 +410,7 @@ services:
     ports:
       - "${PAYMENT_SERVICE_PORT}"
     environment:
+      - FEATURE_FLAG_GRPC_SERVICE_ADDR
```

Comment:
We also need to add this env var to [docker-compose.minimal.yml](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docker-compose.minimal.yml).

Reply:
Good catch, thanks!